Dockerfile

```diff
@@ -1,31 +1,47 @@
+# Stage 1 build
 FROM golang:1.16.1-buster AS builder
+RUN apt-get update && apt-get install ca-certificates -y
+ENV CGO_ENABLED=0
+
+ENV USER=bubbly
+ENV UID=10001
+RUN adduser \
+    --disabled-password \
+    --gecos "" \
+    --home "/nonexistent" \
+    --shell "/sbin/nologin" \
+    --no-create-home \
+    --uid "${UID}" \
+    "${USER}"
 
-WORKDIR /src
+WORKDIR $GOPATH/src/main/bubbly/
 
 COPY go.sum .
 COPY go.mod .
 
 RUN go mod download
-
-RUN apt-get update && apt-get install ca-certificates -y
+RUN go mod verify
 
 COPY . .
 
 RUN go get -u github.com/swaggo/swag/cmd/swag
 RUN swag init
 
-# compile the go binary
-RUN go build -o /bubbly
+RUN GOOS=linux GOARCH=amd64 go build -ldflags="-w -s" -o /go/bin/bubbly
```

Comment:
I _suspect_ that `/go` is `$GOPATH` here, judging by the placement of source (my comment above).